clusters/build-clusters/01_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: vector-audit-log
+  namespace: api-audit-log
+  labels:
+    k8s-app: vector-audit-log
+spec:
+  selector:
+    matchLabels:
+      k8s-app: vector-audit-log
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        k8s-app: vector-audit-log
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      dnsPolicy: Default
+      containers:
+      - name: vector-audit-log
+        image: docker.io/timberio/vector:0.8.2-alpine
+        env:
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: aws-ci-infra-ci-build01-audit-logs-uploader-credentials
+              key: AWS_ACCESS_KEY_ID
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: aws-ci-infra-ci-build01-audit-logs-uploader-credentials
+              key: AWS_SECRET_ACCESS_KEY
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 100m
+            memory: 300Mi
+        volumeMounts:
+        - name: varlog
+          mountPath: /host/var/log
+          readOnly: true
+        - name: config-volume
+          mountPath: /etc/vector
+          readOnly: true
+        - name: cache-volume
+          mountPath: /var/lib/vector
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      serviceAccountName: logging-app
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      terminationGracePeriodSeconds: 30
+      volumes:
+      - name: varlog
+        hostPath:
+          path: /var/log
+      - name: config-volume
+        configMap:
+          name: vector-audit-log-config
+      - name: cache-volume
+        emptyDir: {}
```

Comment:
err...This holds the markers or not? So we definitely want this to persist across pod cycles?

Reply:
I understand the question.

It seems they are doing the same thing.
https://github.com/timberio/vector/blob/6b95f7e17e4af25c689a108cd3fa6941d97d5593/config/kubernetes/vector-daemonset.yaml#L91-L92

Does hostPath sound good?

Reply:
Lets say ... we still use `emptydir`, we could end up with uploading the same log twice.
Anything else?
They are on the masters and they should be stable.
I think `emptydir` shouldnt sound that horrible.

Reply:
yes, hostPath would make sense here imho

Reply:
I dont understand your second comment. Whats the problem with using a hostPath?

Reply:
Done.
Also checked on a master `/var/lib` exists.